web/src/actions/aria.js

```diff
@@ -0,0 +1,8 @@
+export const ARIA_ANNOUNCE_CHANGE = 'ARIA_ANNOUNCE_CHANGE';
+
+export const ariaAnnounceFFPQuarterly = (aKey, year, q, name) => (dispatch, getState) => {
+    dispatch({
+        type: ARIA_ANNOUNCE_CHANGE,
+        message: getState().budget.activities[aKey].quarterlyFFP[year][q][name].dollars
```

Comment:
Should we open an issue for figuring out exactly what this message should be? Right now it reads the new quarterly FFP total but doesn't tell you that's what it is. I feel like that satisfies the original issue but now it's about figuring out what information is useful to the user?

Reply:
Ooh, good call. Yeah, without the `Dollars` component, this doesn't even tell you it's a dollar amount.